homework.md

```diff
@@ -92,13 +92,13 @@ A place for us to store and share our code.
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+This is when I save the changes I made.
```

Comment:
> This is when I save the changes I made.
>
This is correct **but**, I would advise that you be more specific. You wondering what do I mean.

#### What I mean is being specific in the sense that: 

- "Committing is saving all your file changes to your **local repository**"

## Key is where are this commits being saved hope that made sense but Great Answer.